distributed/wiki/app.py

```diff
@@ -0,0 +1,23 @@
+from jina import Flow, Document, DocumentArray
+
+f = Flow.load_config(
+    '/Users/bo/Documents/work/stress-test/distributed/wiki/local/index.yml'
+)
+
+d1 = Document(id=1, text='foo1 is foo fool full fu')
+d2 = Document(id=2, text='foo2 is foo fool full fu')
+d3 = Document(id=3, text='foo3 is foo fool full fu')
+
+
+def print_matches(req):  # the callback function invoked when task is done
+    for idx, d in enumerate(req.docs[0].matches[:3]):  # print top-3 matches
+        print(f'[{idx}]{d.score.value:2f}: "{d.text}"')
+
+
+with f:
+    f.index(inputs=DocumentArray([d1, d2, d3]))
+
+with Flow.load_config(
```

Comment:
Do we need this file in the PR? Looks like some local test to me which will not be used in the actual stress test? If you think its handy to have as some local test entry point, maybe generalize this and otherwise remove?

Reply:
@jacobowitz this will be removed after flow runs successfully, only for test purpose

Reply:
thx for your review.
This PR is still in local progress.